src/components/SearchBar/SearchBar.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { Box, TextField, IconButton } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const SearchContainer = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  width: '324px',
+  height: '67px',
+  left: '929px',
+  top: '91px',
+  background: 'rgba(250, 250, 250, 0.75)', // Fondo con opacidad
+  backdropFilter: 'blur(10px)', // Añadir blur para efecto vidrio
+  borderRadius: '15px',
+  display: 'flex',
+  alignItems: 'center',
+  boxSizing: 'border-box',
+  border: '1px solid rgba(255, 255, 255, 0.2)', // Borde sutil
+
+  [theme.breakpoints.down('sm')]: {
+    width: '342px',
+    height: '67px',
+    left: '24px',
+    top: '231.87px',
+  },
+}));
+
+export const SearchIconButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  width: '19px',
+  height: '19px',
+  left: '31px', // 960 - 929 = 31px desde el contenedor
+  top: '24px', // 115 - 91 = 24px desde el contenedor
+  padding: 0,
+  minWidth: 0,
+  minHeight: 0,
+
+  '&.MuiIconButton-root': {
+    padding: 0,
+    margin: 0,
+    borderRadius: 0,
+  },
+
+  '& .MuiTouchRipple-root': {
+    display: 'none',
+  },
+
+  '&:hover': {
+    backgroundColor: 'transparent',
+  },
+
+  '& img': {
+    width: '17.81px',
+    height: '17.81px',
+    display: 'block',
+  },
+
+  [theme.breakpoints.down('sm')]: {
+    width: '20.06px',
+    height: '19px',
+    left: '32.72px', // 56.72 - 24 = 32.72px desde el contenedor
+    top: '24px', // 255.87 - 231.87 = 24px desde el contenedor
+
+    '& img': {
+      width: '18.8px',
+      height: '17.81px',
+    },
+  },
+}));
+
+export const StyledInput = styled(TextField)(({ theme }) => ({
```

Comment:
StyledInput is absolutely positioned and uses many hard-coded font sizes, line-heights and widths instead of theme.typography and spacing. The CDD rules require typography to come from theme.typography and spacing from theme.spacing.

**Sugerencia:**
```typescript
Let the TextField be layout-managed by its container (no absolute). Use theme.typography for fontFamily/fontSize/lineHeight (e.g. ...theme.typography.body1) and theme.spacing for sizes. Avoid embedding custom fonts inline here; set them at the theme level. If a custom font is required, register it in global styles and reference theme.typography.
```